core/domain/user_jobs_one_off.py

```diff
@@ -81,6 +81,41 @@ def reduce(key, version_and_exp_ids):
                     edited_exploration_ids))
 
 
+class UserAuthModelOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for creating and populating UserAuthModel for
+    all registered users.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(item):
+        """Implements the map function for this job."""
+        user_id = item.id
+        gae_id = [item.gae_id]
```

Comment:
``gae_ids`` since given that this is a list, it's plural in nature.

Reply:
But I think we shouldn't be using multiple gae_ids because we do not have the ability to map multiple email ids to a single account right now

Reply:
gae_id has been updated to single value attribute in PR #10004 and hence changed here also now.

Reply:
Thanks for updating.